dom-event-delegation/main.js

```diff
@@ -0,0 +1,9 @@
+document.addEventListener('click', function (event) {
+  console.log('event.target', event.target);
+  console.log('event.target.tagName', event.target.tagName);
+  if (event.target && event.target.nodeName === 'BUTTON') {
+    const $taskListItem = event.target.closest('.task-list-item');
+    console.log('closest', $taskListItem);
+    $taskListItem.remove();
+  }
+});
```

Comment:
The `addEventListener` should be on your `ul` rather than the `document`. By putting it on the `document`, this will fire anytime you click anywhere on the page. We only want this to fire when we specifically click somewhere in our `ul`

Reply:
That makes sense, code has been updated to query the UL and using that now.